backend/fleet_management/factories.py

```diff
@@ -139,24 +139,26 @@ def drives(self, create, extracted, **kwargs):
                 self.drives.add(drive)
 
 
-class DriveFactory(factory.DjangoModelFactory):
+class DriveFactory(DjangoModelFactory):
 
     class Meta:
         model = Drive
 
-    driver = factory.SubFactory(UserFactory)
-    project = factory.SubFactory(ProjectFactory)
-    car = factory.SubFactory(CarFactory)
+    driver = SubFactory(UserFactory)
+    project = SubFactory(ProjectFactory)
+    car = SubFactory(CarFactory)
     date = fuzzy.FuzzyDate((now() - timedelta(days=1000)).date())
     start_mileage = fuzzy.FuzzyInteger(1000000)
-    description = factory.Faker('text', max_nb_chars=1000)
+    description = Faker('text', max_nb_chars=1000)
     timestamp = fuzzy.FuzzyInteger(1, 999999999)
+    start_location = fuzzy.FuzzyChoice(CITIES)
```

Comment:
I would assume, that's the only reason of these changes.
We should use Faker here also. I would assume that it goes like:
```
start_location = Faker("city", locale="uk_UA")
```

Reply:
Looks good - let me update it

Reply:
Updated - worked perfectly!